### ISSUE TITLE
'Find a student' doesn't list unpublished assignments

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1104]
The `Find a student` feature that got introduced in 3.2 doesn't list unpublished assignments. Searching for a student's assignment could therefore give the impression that the student isn't added to any subjects, when in reality there just hasn't been published any assignments yet.

Probably want to make the unpublished assignments distinguishable from published.

**Comment by StianJul** [comment]
Unpublished assignments are not excluded and I'm not able to reproduce it, but this may be because:
 - The admin user is not an administrator on the semester or subject
 - The student is not added to any assignments on the semester

**Comment by torgeirl** [comment]
I were able to reproduce it with my account (superuser) before an assignment got published so I don't think it's the first thing, and there definitely was one unpublished assignment the student was added to. :confused:

**Comment by StianJul** [comment]
Apparently, there is something else causing it. I guess the assignment is published now, and you can see the assignment for that user through the "Find a student"?

**Comment by torgeirl** [comment]
Correct, once the assignment got published the student could be found using the tool.
